python/ray/autoscaler/v2/tests/test_e2e.py

```diff
@@ -20,6 +20,73 @@ def is_head_node_from_resource_usage(usage: Dict[str, float]) -> bool:
     return False
 
 
```

Comment:
Can you also enable tests in  `test_max_replicas_per_node.py` that's currently disabled due to this issue